### PR TITLE
[RFR] Add readme

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

**Review by Kmaschta — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+For a more thorough introduction see these two nice articles:
+
+- [introduction](https://medium.com/@herrjemand/introduction-to-webauthn-api-5fd1fb46c285)
+- [verifying fido2 responses](https://medium.com/@herrjemand/verifying-fido2-responses-4691288c8770)
+
+## Installation
+
+```js
+npm install webauthn
```
Comment (Kmaschta) [inline README.md:14]: I'm not sure this can work since there is two packages : server and client

**Review by Kmaschta — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+```js
+import { 
+    registrationChallengeToPublicKey, 
+    loginChallengeToPublicKey, 
+    publicKeyToJSON 
+} from '@webauthn/client';
+```
+
+- registrationChallengeToPublicKey:
```
Comment (Kmaschta) [inline README.md:31]: For each function (server + client), can you describe the API of them?
What is a challenge, what is a public key?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- [introduction](https://medium.com/@herrjemand/introduction-to-webauthn-api-5fd1fb46c285)
+- [verifying fido2 responses](https://medium.com/@herrjemand/verifying-fido2-responses-4691288c8770)
+
+## Installation
+
+```js
+npm install webauthn
+```
+
+## usage
```
Comment (Kmaschta) [inline README.md]: s/usage/Usage/

File: README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Webauthn is composed of two parts @webauthn/client and @webauthn/server
+
+### On the browser
```
Comment (Kmaschta) [inline README.md:21]: s/On the browser/On the Browser/

File: README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+- registrationChallengeToPublicKey:
+    convert the challenge returned by the server on the register route into a publicKey
+- publicKeyToJSON:
+    convert the publicKey to JSON.
+- loginChallengeToPublicKey:
+    convert the challenge returned by the server on the login route into a publicKey
+
+See an example in example/front
+
+### On the server
```
Comment (Kmaschta) [inline README.md:40]: s/On the server/On the Server/

Comment (ThieryMichel) [reply README.md:40]: why, it's not a proper noun

File: README.md (hunk 4 above)
Comment (ThieryMichel) [reply README.md:21]: why ?

File: README.md (hunk 2 above)
Comment (ThieryMichel) [reply README.md:31]: I will change the whole api in the next PR

File: README.md (hunk 3 above)
Comment (ThieryMichel) [reply README.md]:
```suggestion
## Usage
```

**Comment by djhi** [comment]
Is this pr still relevant?

**Comment by ThieryMichel** [comment]
Nope you merged the modification with #3

**Pull request closed without merging**